common/src/main/java/com/revenuecat/purchases/common/verification/SigningManager.kt

```diff
@@ -43,39 +41,56 @@ class SigningManager(
             warnLog("Forcing signing error for request with path: $urlPath")
             return VerificationResult.FAILED
         }
-        val signatureVerifier = signatureVerificationMode.verifier ?: return VerificationResult.NOT_REQUESTED
+        val intermediateSignatureHelper = signatureVerificationMode.intermediateSignatureHelper
+            ?: return VerificationResult.NOT_REQUESTED
 
-        if (signature == null) {
+        if (signatureString == null) {
             errorLog(NetworkStrings.VERIFICATION_MISSING_SIGNATURE.format(urlPath))
             return VerificationResult.FAILED
         }
         if (requestTime == null) {
             errorLog(NetworkStrings.VERIFICATION_MISSING_REQUEST_TIME.format(urlPath))
             return VerificationResult.FAILED
         }
-
-        val signatureMessage = getSignatureMessage(responseCode, body, eTag)
-        if (signatureMessage == null) {
+        if (body == null && eTag == null) {
```

Comment:
I'm not sure if this check is needed, but it makes sense to me.

Reply:
Yeah I didn't do this but it makes sense.